src/LocalDate.js

```diff
@@ -12,11 +10,21 @@ function warn(message) {
 
 export default class LocalDate extends Date {
 
+  static ISO_DATE_FORMAT = /^(\d{4})-(\d{2})-(\d{2})$/
+
+  static match(isoDate) {
```

Comment:
@giogonzo maybe we should call this `test` to reflect `RegExp` API. What do you think?